rts/at/sra/README.md

```diff
@@ -0,0 +1,68 @@
+OVERVIEW
+--------
+An example of how to use REST-AT service.
+
+USAGE
+--------------------
+TEST SRA PARTICIPANTS
+
+Start a REST-AT coordinator on port 8080
+
+    cd <narayana-repo>/jboss-as/build/target/wildfly-28.0.0.Final
+    cp docs/examples/configs/standalone-rts.xml standalone/configuration
+    ./bin/standalone.sh -c standalone-rts.xml
+
+VERIFY THAT IT IS RUNNING, eg:
+
+    curl http://localhost:8080/rest-at-coordinator/tx/transaction-manager
+
+BUILD  AND RUN THE DEMO APPLICATION on port 8081
+
+    cd <narayana-repo>/rts/sra
+    mvn clean package -Dquarkus.package.type=uber-jar
+    java $JAVA_OPTS -Dquarkus.http.port=8081 -jar target/sra-participant-runner.jar &
+    
+    OR
+    mvn quarkus:dev
+
+Book a trip within a transaction (see sra/demo/api/TripController.java method bookTrip)
+this starts a transaction and you can verify it is running using
+curl http://localhost:8080/rest-at-coordinator/tx/transaction-manager
+
+    curl -XPOST http://localhost:8081/trip/book?hotelName=Rex
+
+EXPECTED OUTPUT
+---------------
+    SRA: 0_ffff0a4cf0b6_-568c278f_6458bef8_7fb: Updating hotel participant state to: TransactionPrepared
+
+    SRA: 0_ffff0a4cf0b6_-568c278f_6458bef8_7fb: Updating trip participant state to: TransactionPrepared
+
+    SRA: 0_ffff0a4cf0b6_-568c278f_6458bef8_7fb: Updating trip participant state to: TransactionCommitted
+
+    SRA: 0_ffff0a4cf0b6_-568c278f_6458bef8_7fb: Updating hotel participant state to: TransactionCommitted
+
```

Comment:
@marcosgopen I recall that you tested the demo - do you recall whether or not this was the output?

Reply:
Yes, I confirm that is the output. Thanks to @moulalis changes there are new lines now. And thanks the previous Moulali PR now state is 'TransactionCommitted' (which is correct)

Reply:
This is updated, so resolving the conversation.